internal/enginecache/version.go

```diff
@@ -23,3 +32,102 @@ func flutterRequiredEngineVersion() string {
 
 	return versionMatch[1]
 }
+
+// CheckFoGoFlutterUpdate check the last 'go-flutter' timestamp we have cached
+// for the current project. If the last update comes back to more than X days,
+// fetch the last Github release semver. If the Github semver is more recent
+// than the current one, display the update notice.
```

Comment:
How much time would it cost to fetch the GitHub release semver on a bad connection?
I would really prefer to increase how often people check for an update (rather 1 hour instead of 2 days), but that's only feasible if checking for the release semver on github doesn't cost too much time (ofc it's already way faster than always go-getting, which is awesome!!).